verify/index.ts

```diff
@@ -47,9 +47,7 @@ async function configTrustStore(dir: string) {
             let trustStore = trustStores[j]; // .github/truststore/x509/ca/<my_store>
             let trustStoreName = path.basename(trustStore); // <my_store>
             let certFile = getFileFromDir(trustStore); // [.github/truststore/x509/ca/<my_store>/<my_cert1>, .github/truststore/x509/ca/<my_store>/<my_cert2>, ...]
-            for (const cert of certFile) {
-                await exec.getExecOutput(`notation cert add -t ${trustStoreType} -s ${trustStoreName} "${cert}"`);
-            }
+            exec.getExecOutput(`notation cert add -t ${trustStoreType} -s ${trustStoreName}`, certFile);
```

Comment:
Can we keep the first parameter to `notation` only? Any [UGC](https://en.wikipedia.org/wiki/User-generated_content) can lead to security vulnerabilities.